src/js/msp/MSPConnector.js

```diff
@@ -1,4 +1,5 @@
-'use strict';
+import MspHelper from "./MSPHelper";
+import { read_serial } from "../serial_backend";
```

Comment:
@haslinghuis this should fix firmware flasher issue.

Reply:
Thanks.